src/components/Match/matchPages.jsx

```diff
@@ -45,45 +45,39 @@ const filterMatchPlayers = (players, team = '') =>
     ((team === 'radiant' && isRadiant(player.player_slot)) || (team === 'dire' && !isRadiant(player.player_slot)) || team === '')
   ).sort((a, b) => a.player_slot - b.player_slot);
 
-const MatchPlayersTable = ({
-  match,
-  columns,
-}) => (<Table data={match.players} columns={columns} />);
-
-const MatchPlayersTableSplit = ({
+const TeamTable = ({
   match,
   columns,
+  heading = '',
 }) => (<div>
+  <Heading title={`${strings.general_radiant} ${heading}`.trim()} />
   <Table data={filterMatchPlayers(match.players, 'radiant')} columns={columns} />
+  <Heading title={`${strings.general_dire} ${heading}`.trim()} />
   <Table data={filterMatchPlayers(match.players, 'dire')} columns={columns} />
 </div>);
 
+// TODO: Add networth and xp graph to overview
 const matchPages = [{
   name: strings.tab_overview,
   content: match => (<div>
-    <Heading title={strings.heading_overview} />
-    <MatchPlayersTableSplit match={match} columns={overviewColumns(match)} />
-    <Heading title={strings.heading_ability_builds} />
-    <MatchPlayersTable match={match} columns={abilityUpgradeColumns} />
+    <TeamTable match={match} columns={overviewColumns(match)} />
```

Comment:
I like this name because once we implement the option to merge players, this function will produce a single table and not split tables. I wanted to name it `Table` but didn't want to mess with existing component names.